go-app/templates/events.html

```diff
@@ -1,33 +1,162 @@
-
 <!DOCTYPE html>
 <html>
 <head>
-  <meta charset="utf-8" >
-  <title> Radar-log-parser</title>
- <link rel="stylesheet" href="/assets/styles.css">
- <style>
- </style>
+    <meta charset="utf-8" >
+    <title> Radar-log-parser</title>
+   <link rel="stylesheet" href="/assets/styles.css">
+   <style>
+     .details{
+        display: none;
+     }
+    
+     .banner_after,.banner_before{
+        width:100%;
+        padding:0.5%;
+        background-color: gray;
+        margin-bottom: 0.5%;
+     }
+     span{
+        cursor: pointer;
+        margin-left:10%;
+     }
+     #main{
+        margin-left:40%;
+     }
+    .event {
+        color: fuchsia;
+        cursor: pointer; 
+        text-align: left;
+        text-decoration: underline;
+     }
+  </style>
+  <script> 
+   function expandContent(element){
+     var content_elem = element.parentNode.nextElementSibling;
+     var expand_number = element.innerHTML;
+     if (expand_number == "Block"){
+        content_elem.style.display = "none";
+        return;
+     }
+     if (content_elem.style.display === "none") {
+        content_elem.style.display = "block";
+     }
+     var line_number = parseInt(content_elem.getAttribute("data-line-number"));
+     if(line_number<0 || line_number>=Math.pow(2, 52)){
+         return;
+     }
+     var size;
+     if(expand_number == "All"){
+        size =  Math.pow(2, 52) ;
```

Comment:
Try to avoid these sorts of arbitrary big numbers when trying to do something like selecting all lines. Use a constant that the language provides if possible (e.g. max int in kotlin https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/-m-a-x_-v-a-l-u-e.html) or do something like make it an optional field and treat a missing value as an indicator to consume all of the content.

Reply:
Hum, Ok. It is actually the value of the constant. It is not an arbitrary number.
But, what do you mean by :"treat a missing value as an indicator to consume all of the content."